### PR TITLE
improved snackbar visually, introduced a component as a snackbar

### PR DESCRIPTION
**Pull request by KalaRaja**

### CODE REVIEW

File: src/app/services/app-updater.service.ts [hunk 1]
```diff
@@ -4,12 +4,14 @@ import { SwUpdate } from '@angular/service-worker';
...
 
+import { SnackbarComponent } from '../snackbar/snackbar.component';
+
 @Injectable({
   providedIn: 'root'
 })
 export class AppUpdaterService {
   isSnackbarloaded = false;
-  checkUpdateInterval = 1000 * 60 * 2 * 60; // 2 hours
+  checkUpdateInterval = 1000; // 2 hours
```
Comment (bherr2) [inline src/app/services/app-updater.service.ts]: That doesn't look like two hours...

File: src/app/snackbar/snackbar.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+import { AppUpdaterService } from '../services/app-updater.service';
+
+@Component({
+  selector: 'mav-snackbar',
+  templateUrl: './snackbar.component.html',
+  styleUrls: ['./snackbar.component.scss']
+})
+export class SnackbarComponent {
```
Comment (bherr2) [inline src/app/snackbar/snackbar.component.ts]: Why is this generically called Snackbar Component? Shouldn't it be something like UpdateNotificationComponent or something along those lines?

File: src/app/services/app-updater.service.ts (hunk 1 above)
Comment (KalaRaja) [reply src/app/services/app-updater.service.ts]: oops! will fix it

File: src/app/app-update-notification/app-update-notification.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AppUpdateNotificationComponent } from './app-update-notification.component';
+
+describe('SnackbarComponent', () => {
```
Comment (bherr2) [inline src/app/app-update-notification/app-update-notification.component.spec.ts]: Missed a spot :-)

File: src/app/services/app-updater.service.ts [hunk 4]
```diff
@@ -46,18 +48,23 @@ export class AppUpdaterService {
   /**
    * Opens snack bar.
    * Uses angular material snackbar service to open the snackbar.
-   * When 'Click here to update!' button is pressed, updateApp function is called.
+   * When 'REFRESH' button is pressed, updateApp function is called.
    */
   openSnackBar() {
-    this.snackbar.open('New update available', 'Click here to update!', {
+    this.snackbar.openFromComponent(AppUpdateNotificationComponent, {
       horizontalPosition: 'center',
```
Comment (bherr2) [inline src/app/services/app-updater.service.ts]: Move to bottom, left

**Pull request merged**